addons/website_event_track/views/event_track_templates_list.xml

```diff
@@ -52,7 +52,7 @@
                     </a>
                 </div>
             </div>
-            <div class="o_wevent_search d-flex w-100 w-lg-auto">
+            <div class="o_wevent_search o_wevent_event_track_search_box d-flex w-100 w-lg-auto">
```

Comment:
Hello @thsh-odoo 

Since we are in master and have some liberty on changes, would it be possible to instead modify the XML structure to comply with what the "searchBar" snippet expects?

That way we have a standard component and avoid a JS override?

Thanks for looking into it & cheers!